python/lsst/ts/Dome/dome_csc.py

```diff
@@ -21,47 +21,65 @@ def __init__(
         self.writer = None
         self.config = None
 
+        # Tasks that are run to execute specific functionality.
+        self.status_task = salobj.make_done_future()
+        self.connect_task = salobj.make_done_future()
+
         self.mock_ctrl = None  # mock controller, or None if not constructed
         super().__init__(
-            name="DCS",
+            name="Dome",
             index=0,
             schema_path=schema_path,
             config_dir=config_dir,
             initial_state=initial_state,
             simulation_mode=simulation_mode,
         )
 
-    async def do_crawlAz(self, data):
+    async def do_moveAz(self, data):
         """
-        Crawl AZ
+        Move AZ
```

Comment:
Really pick point, but please put the brief description on the same line as the opening `"""`. In general we follow DM's convention for Python doc strings: https://developer.lsst.io/python/numpydoc.html (and DM basically follows numpy's convention).

Reply:
Done